keps/sig-scheduling/2779-support-custom-queue-for-the-scheduler/kep.yaml

```
@@ -0,0 +1,42 @@
+title: support custom queue for the scheduler
+kep-number: 2779
+authors:
+  - "@suigh"
+owning-sig: sig-scheduling
+status: provisional
+creation-date: 2021-06-09
+reviewers:
+  - "@Huang-Wei"
+  - TBD
+approvers:
+  - "@Huang-Wei"
+  - TBD
+
+see-also:
+  - "/keps/sig-scheduling/624-scheduling-framework"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.22"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.22"
+  beta: "v1.23"
+  stable: "v1.24"
```

We won't be able to finish in alpha in this release:

```suggestion
  alpha: "v1.23"
  beta: "v1.24"
  stable: "v1.26"
```